Shared/Samples/Create mobile geodatabase/CreateMobileGeodatabaseView.Model.swift

```diff
@@ -0,0 +1,215 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension CreateMobileGeodatabaseView {
+    /// The view model for the sample.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// A map with a topographic basemap centered on Harpers Ferry, West Virginia, USA.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTopographic)
+            map.initialViewpoint = Viewpoint(latitude: 39.3238, longitude: -77.7332, scale: 1e4)
+            return map
+        }()
+        
+        /// A URL to a temporary geodatabase.
+        private let geodatabaseURL: URL
+        
+        /// A URL to a temporary directory to store the geodatabase.
+        private let directoryURL: URL
+        
+        /// The mobile geodatabase.
+        private var geodatabase: Geodatabase?
+        
+        /// The feature table in the geodatabase.
+        private var featureTable: GeodatabaseFeatureTable?
+        
+        /// The table description used to create a new feature table in the geodatabase.
+        private let tableDescription: TableDescription = {
+            // Create a description for the feature table.
+            let description = TableDescription(
+                name: "LocationHistory",
+                spatialReference: .wgs84,
+                geometryType: Point.self
+            )
+            
+            // Create and add the field descriptions for the table.
+            description.addFieldDescriptions([
+                // `FieldType.OID` is the primary key of the SQLite table.
+                FieldDescription(name: "oid", fieldType: .oid),
+                // `FieldType.DATE` is the date column used to store a calendar date.
+                FieldDescription(name: "collection_timestamp", fieldType: .date)
```

Comment:
OID and oid seem to match but DATE and collection_timestamp do not. Should one change to match the other?

Reply:
To my knowledge [FieldType](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/fieldtype) doesn't have a `timestamp` property, so we are using the `date` property as a timestamp here. The old iOS version did [the same thing](https://github.com/Esri/arcgis-runtime-samples-ios/blob/main/arcgis-ios-sdk-samples/Edit%20data/Create%20mobile%20geodatabase/CreateMobileGeodatabaseViewController.swift#L138)  as well as all the other implemenations.